manifest.php

```diff
@@ -52,6 +55,8 @@
             RegisterEncryptionSymmetricService::class,
             RegisterEncryptResultStorage::class,
             RegisterDecryptResultStorage::class,
+            RegisterSimpleKeyProviderService::class,
```

Comment:
This one is not usefull, SimpleKeyProvider have not options and FileKeyProvider will override it

Reply:
no necessary